src/Tribe/Admin/Settings.php

```diff
@@ -5,14 +5,12 @@
  * Manages the admin settings UI in relation to events configuration.
  */
 
+use Tribe\Admin\Troubleshooting as Troubleshooting;
 use Tribe__App_Shop;
-use Tribe__Settings;
+use Tribe__Events__Main as Plugin;
```

Comment:
You shouldn't need this for the `tribe()` call later

Reply:
I just sorted the `use` statements.